bevy_asset_loader/Cargo.toml

```diff
@@ -13,8 +13,16 @@ categories = ["game-development"]
 readme = "../README.md"
 
 [features]
-# This feature requires bevy's TextureAtlas and StandardMaterial assets to be available ("bevy/render")
-render = ["bevy_asset_loader_derive/render"]
+# This feature adds support for bevy's TextureAtlas assets
+2d = [
+    "bevy/bevy_sprite",
```

Comment:
Do we actually need the Bevy features?

Reply:
Well, I don't know (haven't tested) if we *strictly need* them (as in, can we compile without them), but the relevant functionality that our features enable is kinda useless without them anyway. I think there shouldn't be a situation where the user wants to enable the `2d`/`3d` on this crate, but disable the bevy features. I decided to add them here to improve the UX, to ensure bevy is always built appropriately / the user hasn't forgotten about them.

Reply:
Sounds good to me :+1: